docs/cep/table/index.md

```diff
@@ -0,0 +1,72 @@
+---
+
+A table is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../collections/index.md).
+
+## Purpose
+
+Tables allow the stream worker to work with stored events. By defining a schema for tables, the stream processor enables them to be processed by queries using their defined attributes with the streaming data. You can also query the state of the stored events in the table.
+
+Any table defined in a stream worker is automatically a store for that stream worker. Tables can be stores or [sinks](../sink/).
+
+You can query table contents using [Table Operators](table-operators).
+
+## Syntax
+
+There are two ways to define tables.
+
+### CREATE TABLE
+
+The syntax for a new table definition is as follows:
+
+```sql
+CREATE TABLE (GLOBAL|SPOT)? <table_name> (<attribute_name> <attribute_type>, ...);
+```
+
+For example:
+
+```sql
+CREATE TABLE SensorTable (sensorId string, temperature double);
+```
+
+### CREATE STORE
+
+You can also use general store syntax:
+
+```sql
+CREATE STORE (GLOBAL|SPOT)? <table_name> WITH(type="<store_type>", propKey=”propVal”, … , PrimaryKey='<attribute_name>', Index='<attribute_name>')(<attribute_name> <attribute_type>, ...);
+```
+
+For example:
+
+```sql
+CREATE STORE SensorTable WITH(type=’database’, collection=’SampleTable’, map.type=’json’) (sensorId string, temperature double);
+```
+
+## Parameters
+
+The following parameters are configured in a table definition:
+
+| Parameter     | Description |
+| ------------- |-------------|
+| table name      | The name of the table defined. (`PascalCase` is used for table name as a convention.) |
+| attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
+| attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
+| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
+| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+
+## Examples
+
+The following defines a local table named `RoomTypeTable` with `roomNo` and `type` attributes of data types `int` and `string` respectively.
+
+```sql
+CREATE TABLE RoomTypeTable (roomNo int, type string);
+```
+
+The following defines a global table named `SweetProductionCollection` with `name` and `amount` attributes of data types `string` and `double`.
+
+```sql
+CREATE TABLE GLOBAL SweetProductionCollection (name string, amount double);
```

Comment:
@diana-macrometa - All collections a global by default.

Reply:
I thought the default was local. This is how the example was written.

Reply:
@diana-macrometa - It was before the introduction of `play` fed. I cannot remember what was the issue but we/I had to urgently change the default to `GLOBAL`.

Reply:
Here's where I found that. [Stream Sinks](https://www.macrometa.com/docs/cep/sink/sink-types/stream-sink#query-parameters) are apparently local by default. Is it different for tables?

Reply:
Ah, then should I update the stream sink topic as well?

Reply:
@diana-macrometa - The streams are still local by default. Fortunately I succeed to keep this unchanged. :)